### DISCUSSION TITLE
About InvOut and Inv breakpoints

### DISCUSSION THREAD

**Discussion by wen10-git** [schneebergerlab/syri#243] [Q&A]
Hello, @mnshgl0110 
First of all this is a very good software.
Then my question is about the InvOut.txt file. I noticed that the file is in this format. The components after "#" are easy to understand, as they are the starting point and end point of the inversion, but how to understand the intermittent sequence below? Are they genes? Is the part between these sites a breakpoint?

![image](https://github.com/schneebergerlab/syri/assets/63379795/6806564a-71b6-487f-b4f0-f85e4b011830)

Then I discovered that the NotAli.txt file inclusion location didn't seem to include the location within InvOut.txt. I was expecting to be able to find the "inversion breakpoint" with NotAli.txt. The reason for this is that in the paper we submitted, the reviewer believed that the position of the "inversion breakpoint" between a pair of genomes may determine one's environmental adaptability. I don't seem to find any clear answers to similar questions in the Issues library. 

Thanks!.

**Comment by wen10-git** [comment]
Hey, I'm sorry can you reply to me? @mnshgl0110 
Your reply is very important to me!
I saw a reply from earlier. https://github.com/schneebergerlab/syri/issues/232 I'm not sure if the small "start-end" interval below a large "invert" is what is called a "breakpoint"?
![image](https://github.com/schneebergerlab/syri/assets/63379795/c8d7a65c-a728-4fbc-ad1c-e33a5346adec)

**Comment by mnshgl0110** [comment]
Hi @Wenwen012345. Sorry for a delayed response, I was busy with other stuff.

I would recommend that you use and analyse the syri.out file which is described [here](https://schneebergerlab.github.io/syri/fileformat.html).

Large inversions can consists of multiple alignments (check Fig S2 from the syri manuscript). In the invOut.txt, lines without '#' corresponds to the coordinates of these alignments.

Typically, inversion breakpoints refer to the start and end of the entire inversion (that is the coordinates described by lines with '#').

**Comment by wen10-git** [comment]
Hi @mnshgl0110 
Thank you very much for your reply, very useful information!

Now I have a few more questions that you might answer if it's convenient for you. Excuse me.
First of all the reviewer's question to us was:
"Secondly, in comparison to variations in gene numbers, structural variations, especially genome inversions, have been shown to play crucial roles in environmental adaptation during plant evolution. Could insights into the adaptation of R. bailiense to the karst landform be provided by analyzing the functions of genes associated with structural variations, such as those involved in inversion breakpoints?"


From this question, I would like to ask:
**1) Do Not_Aligned regions considered  "structural variations"?
2) In your experience, what kind of variation is most likely to cause trait differences between two closely related species? For example, is it the "inversion breakpoint"? What are the most important aspects we should focus on?**
Sincerely!
Wen

**Comment by mnshgl0110** [comment]
> 1) Do Not_Aligned regions considered "structural variations"?

Yes. Syri does consider them as SVs but this is a subjective question and different people would have different opinions.

>  2) In your experience, what kind of variation is most likely to cause trait differences between two closely related species? For example, is it the "inversion breakpoint"? What are the most important aspects we should focus on?

I don't think there is any ranking describing which variation is more important. Typical analysis involves looking at the trait of interest and associated variation.
